fluentd/run.sh

```diff
@@ -142,10 +142,11 @@ else
     if [ "$ES_HOST" = ${OPS_HOST:-""} -a $ES_PORT -eq ${OPS_PORT:-0} ]; then
         # There is one output Elasticsearch
         NUM_OUTPUTS=1
-        # Disable "output-es-ops-config.conf in output-operations.conf"
-        echo > $CFG_DIR/dynamic/output-es-ops-config.conf
+        # Disable "output-operations.conf"
+        rm -f $CFG_DIR/openshift/output-operations.conf
+        touch $CFG_DIR/openshift/output-operations.conf
         rm -f $CFG_DIR/openshift/filter-post-z-retag-*.conf $CFG_DIR/openshift/filter-post-mux-client.conf
-        if [ "$output_label" != "" ]; then
+        if [ -n "$output_label"  ]; then
```

Comment:
There are 2 more 'if [ "$output_label" != "" ]'.  If you could replace them with 'if [ -n "$output_label"  ]', I'd appreciate it.